_data/packages.yml

```diff
@@ -617,15 +654,15 @@
     - data-processing-munging
   editor:
       name: Emily Jane McTavish
-      github_username: snacktavish
+      github_username: snacktavish Emily Jane McTavish
```

Comment:
This is another, different bug that recently emerged. why?
here the name is correct it's just released. we should have a validation step on the github username that it can only be one word. if there is a second word it should only include the first word. github usernames can't have spaces so this should be easily to validate in our model.